### ISSUE TITLE
Error: "passwd" No such file or directory

### ISSUE THREAD

**Issue by anonymouso5310** [BlackArch/blackarch-installer#73]
Hello 
I tried to install blackarch 2021.09.01 on my pc but when installing the packages I got this error
(chroot failed to run command "passwd"  No such file or directory)

**Comment by anonymouso5310** [comment]
I opted for root and i use blackarch netinstall

**Comment by yawns** [comment]
tl;dr: Run `pacman -Syy archlinux-keyring` and `pacman -Syy blackarch-installer` before starting the install.

I was having the same problem ("chroot: failed to run passwd" spamming the screen). I recorded my installation and played it back to see if any errors were flashing by. Turns out it's doing this:

![Screenshot 2022-01-26 104531](https://user-images.githubusercontent.com/2416785/151196899-2daa0b78-617e-4c4f-905e-eaa5a1b7cc58.png)

So it's just rejecting the entire base install. After this there's a lot of "no such file" errors, but it all gets pushed off the screen because it gets stuck in a loop trying to run passwd.

Fix is to update the keyring and the installer before running the install.

**Comment by anonymouso5310** [comment]
when installing the keyring and the installer I have this
![20220128_183031](https://user-images.githubusercontent.com/98212619/151595098-e998292a-008a-405e-a71c-f91188ad5e86.jpg)
 error 
![20220128_183158](https://user-images.githubusercontent.com/98212619/151595111-def58ea8-b704-4f35-8d99-ddb8a68a15a7.jpg)

**Comment by dualfade** [comment]
Check networking. Is it working ? Does it resolve?

Interface up ?
ip a sh

[0] % host mirror.rackspace.com
mirror.rackspace.com has address 72.4.120.219

Edit your mirrorlist and remove that bad mirrors if you have some.
ls -l /etc/pacman.d/mirrorlist*
